sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -74,7 +74,16 @@ private[hive] object SparkSQLCLIDriver {
       System.exit(1)
     }
 
-    val sessionState = new CliSessionState(new HiveConf(classOf[SessionState]))
+    val localMetastore = {
+      val temp = Utils.createTempDir()
+      temp.delete()
+      temp
+    }
+    val cliConf = new HiveConf(classOf[SessionState])
+    // Override the location of the metastore since this is only used for local execution.
```

Comment:
So, if I understand this correctly, you're creating just a "dummy" local metastore that won't actually be used; this is just to keep the Hive libraries happy, right?